webpack.config.js

```diff
@@ -1,6 +1,11 @@
+import config from 'config';
 import webpack from 'webpack';
 import path from 'path';
 
+const GLOBALS = {
+  API_BASE_URL: JSON.stringify(`${config.api.host}:${config.api.port}${config.api.baseUrl}`)
```

Comment:
What do you think to have this as 2nd option @pbarrios? and the first one the real ENV? 

Something like this:
`JSON.stringify(process.env.API_BASE_URL || `${config.api.host}:${config.api.port}${config.api.baseUrl}`)`

So that in that way it's easier to update the value from a CI, command line, etc. Same functionality but a bit more of power here.